pyface/tests/test_resource_manager.py

```diff
@@ -47,3 +51,41 @@ def __len__(self):
         resource_manager = ResourceManager()
         img_ref = resource_manager.locate_image("core.png", sequence)
         self.assertEqual(IMAGE_PATH, img_ref.filename)
+
+    def test_locate_image_with_module(self):
+        # images/close.png is included in pyface package data.
+        # ResourceManager can find it via package resources
```

Comment:
should "via package resources" be `importlib_resources`".  I guess package resources does not mean `pkg_resources`, it is a more general term, in which case this is good as is

Reply:
Yea I was intending for this to be a more general term, I can also reword this comment to avoid the term.

Reply:
That makes sense, in that case either way is fine with me